main.c

```diff
@@ -56,21 +49,20 @@ int main(int argc, char *argv[]) {
   };
 
   static slip_handler_s slip;
-  struct sp_port *port = NULL;
 
   uint8_t prev_input = 0;
   uint8_t prev_note = 0;
   uint16_t zerobyte_packets = 0; // used to detect device disconnection
 
   signal(SIGINT, intHandler);
   signal(SIGTERM, intHandler);
+  signal(SIGQUIT, intHandler);
```

Comment:
This doesn't work under Windows, it might need to be wrapped in a "#ifdef SIGQUIT"